Description: Add missing includes in configure check for OpenSSL
 With the new -Werror=implicit-function-declaration build flag set by
 dpkg-buildflags the configure check for OpenSSL fails due to a missing
 include for exit(3).
Author: Guillem Jover <gjover@sipwise.com>
Last-Update: 2018-09-12

---
 configure.in |    1 +
 1 file changed, 1 insertion(+)

--- a/configure.in
+++ b/configure.in
@@ -965,6 +965,7 @@ if test "x$ssllib" != "x" && test "x$ssl
                        openssl/hmac.h)
       AC_MSG_CHECKING(whether the OpenSSL library is multithread-enabled)
       AC_TRY_RUN([
+        #include <stdlib.h>
         #define OPENSSL_THREAD_DEFINES
         #include <openssl/opensslconf.h>
         int main(void) {
